community/ffmpegthumbnailer/APKBUILD

```diff
@@ -23,13 +23,13 @@ prepare() {
 }
```

Comment:
You can also remove `prepare` function; applying of patches is already handled by the `default_prepare` function that is called when no custom `prepare` function is declared.